ferrocene/ignored-tests.toml

```diff
@@ -126,3 +126,10 @@ tests = [
 ]
 targets = ["aarch64-unknown-ferrocene.facade", "thumbv7em-ferrocene.facade-eabi", "thumbv7em-ferrocene.facade-eabihf"]
 reason = "Currently output of these tests experiences multiple stdout writers colliding to create unreproducible results"
+
+[["tests/ui"]]
+tests = [
+    "tests/ui/alloc-error/alloc-error-backtrace.rs"
+]
+targets = ["aarch64-unknown-ferrocene.facade", "thumbv7em-ferrocene.facade-eabi", "thumbv7em-ferrocene.facade-eabihf"]
+reason = "These targets cannot unwind"
```

Comment:
I don't think this reason is correct, since the test passes on `aarch64-unknown-none`. Maybe say something like "These fail on the facade targets for an unknown reason."

I'm still not super happy we're disabling this test, I would like to know why it's failing.